wurst/objects/abilities/scout/Trap.wurst

```diff
@@ -19,6 +19,8 @@ import UnitExtensions
 import LocalObjectIDs
 import ToolTipsUtils
 
+import Track
```

Comment:
This sort of dependency is bad as I don't want to worry about circular imports. I would define the track in that file directly and just reference the ability ID. It's messy in a different way, but I think this should be the standard because WC3 already related independent object purely on their ID. It also makes it easier to see the implementation for an ability.